WalletWasabi.Fluent/ViewModels/RecoverWalletPageViewModel.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Reactive.Linq;
+using DynamicData;
+using DynamicData.Binding;
+using NBitcoin;
+using ReactiveUI;
+using Splat;
+using WalletWasabi.Gui;
+using WalletWasabi.Gui.Validation;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.ViewModels
+{
+    public class RecoveryPageViewModel : NavBarItemViewModel
+    {
+        private readonly ObservableAsPropertyHelper<Mnemonic?> _currentMnemonic;
+        private readonly ObservableAsPropertyHelper<bool> _isMnemonicValid;
+        private ObservableCollection<string> _mnemonics;
+        private IEnumerable _suggestions;
+        private int _itemCountLimit;
+        private bool _restrictInputToSuggestions;
+        private object _selectedTag;
+
+        public ObservableCollection<string> Mnemonics
+        {
+            get => _mnemonics;
+            set => this.RaiseAndSetIfChanged(ref _mnemonics, value);
+        }
+
+        public IEnumerable Suggestions
+        {
+            get => _suggestions;
+            set => this.RaiseAndSetIfChanged(ref _suggestions, value);
+        }
+
+        public int ItemCountLimit
+        {
+            get => _itemCountLimit;
+            set => this.RaiseAndSetIfChanged(ref _itemCountLimit, value);
+        }
+
+        public bool RestrictInputToSuggestions
```

Comment:
Remove these properties... no need for us to bind them we dont change them... just set in the XAML.